src/screens/TopicPageScreen/index.js

```diff
@@ -41,6 +41,7 @@ const TopicPageScreen = (props) => {
   const [userId, setUserId] = React.useState('');
   const [topicId, setTopicId] = React.useState('');
   const [isFollow, setIsFollow] = React.useState(params.isFollowing);
+  const [followType, setFollowType] = React.useState('');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of a new state variable `followType` is a significant change. It appears to be used to track the type of following (signed or incognito) for a topic.
```diff
+ const [followType, setFollowType] = React.useState('');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->